src/Data/Vec/Relation/Binary/Pointwise/Inductive.agda

```diff
@@ -232,6 +235,42 @@ module _ {_∼_ : Rel A ℓ} (refl : Reflexive _∼_) where
   cong-[ zero ]≔  p (_   ∷ eqn) = refl ∷ eqn
   cong-[ suc i ]≔ p (x∼y ∷ eqn) = x∼y  ∷ cong-[ i ]≔ p eqn
 
+------------------------------------------------------------------------
+-- zipWith
+
+module _ {_∼_ : Rel A ℓ} where
+  module _ {f : A → A → A} where
```

Comment:
If you're going to go with nested anonymous modules (rather than private variables), then I would also lift `assoc : Associative _∼_ f` to the module signature, so that it doesn't have to be repeated in the code below. Same with all the other modules.

But I'm going to leave this as a suggestion rather than a must-do. I don't think we have fully settled on a style for this yet.

Reply:
For consistency, I followed the style in `Data.Vec.Properties` (https://github.com/agda/agda-stdlib/blob/8c5dee34ebde4734f8da5369c3f735576a6311fc/src/Data/Vec/Properties.agda#L572).

Reply:
Right - consistency wins! Ignore my suggestion for this PR.